MAUI/Cartesian-Charts/Waterfall-Chart.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: Waterfall Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about the Waterfall Series chart and its features in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Waterfall Chart in .NET MAUI Chart
+
+## Waterfall Chart
+
+[WaterfallSeries]() clarifies the cumulative effect of a set of provided positive and negative values. The series is represented by a rectangle and a connector between the rectangles. To render a waterfall chart, create an instance of [WaterfallSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of the [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1)
+
+*`SummaryBindingPath`- gets or sets the string value that indicates the sum of previous segments in the series.
```

Comment:
![image](https://user-images.githubusercontent.com/92786382/225218069-7e890644-0391-4a6b-8eee-2dc7f40c2a72.png)

@SuryaKaran2143 while explaining properties use above format. 

example
[SummaryBindingPath]() of the string type is used to get or set the path value on the source subject to serve summary values to the series.

Reply:
I Solved this issuue